src/pages/KvitteringExcel.tsx

```diff
@@ -1,13 +1,13 @@
 import React from 'react';
-import './Kvittering.less';
-import InnloggetSide from "./InnloggetSide";
 import {MottattSoknad} from "../components/kvittering/MottattSoknad";
+import {Container} from "nav-frontend-grid";
 
 const KvitteringExcel = () => {
   return (
-    <InnloggetSide>
+    // @ts-ignore
+    <Container style={{marginTop: "0.5rem"}}>
```

Comment:
`<Container style={{marginTop: "0.5rem"}}>` => `<Container className='kvittering-excel'}}>` og sett på styling via less

Reply:
Det føltes unødvendig å lage en hel .less-fil for 1 stk style, men det er altså det beste å gjøre uansett?

Reply:
I know, men det er slippery slope å bruke innline styling og det overrider vanlig styling. Så hvis komponenten bliver stor og man har misset å fjerne en innline styling somewhere da kan det være uoverskueligt for ny utvikler og fikse.

Er bedre helt å unngå det :)